src/Likes.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App;
+
+use PDO;
+
+class Likes
+{
+    private PDO $pdo;
+
+    public function __construct()
+    {
+        $this->pdo = DB::connect();
+    }
+
+    public function getLikes(int $id, int $userId): bool
+    {
+        $query = "SELECT * FROM likes WHERE ads_id = :id AND user_id = :userId";
+        $stmt = $this->pdo->prepare($query);
+        $stmt->bindParam(':id', $id, PDO::PARAM_INT);
+        $stmt->bindParam(':userId', $userId, PDO::PARAM_INT);
+
+        $stmt->execute();
+
+        // Tekshirish: agar bitta qator bo'lsa, true qaytar
+        return $stmt->rowCount() > 0;
```

Comment:
If you needed number of row, you could use COUNT() right in you query:
SELECT COUNT(*) AS count ...
// some other code
return $stmt->fetch()->count > 0;